CONTRIBUTING.md

```diff
@@ -140,16 +140,18 @@ Because SQL within JSON or YAML will not generally be syntax highlighted, indent
 
 ## Map Icon Guidelines
 
-Design Guidelines
-* SVG only
-* flat meaning a single colour, no gradients, no outlines
-* common canvas size, which is usually 14x14 px
-* single point of view and avoid use of perspective where possible
-
-Best Practices
-* align to pixel grid
-* clean design, so reduced complexity where possible
-
+* All new icons must be SVG format only.  The SVG must be saved as standards compliant SVG without any prioprietery tags.  In Inksapce software, you will need to Save As... Plain SVG format.
+* Icons must be use fills and not outlines.
+* Use a single color, typically black, for the fill portion of the icon.  MSS (Map Style Sheets) replaces the single color with the appropriate thematic color.
```

Comment:
Not black, but no colour at all. An example is the [parking symbol](https://github.com/gravitystorm/openstreetmap-carto/blob/master/symbols/parking.svg)

```svg
<?xml version="1.0" encoding="utf-8"?>
<svg xmlns="http://www.w3.org/2000/svg" width="100%" height="100%" viewBox="0 0 9 12">
<path fill-rule="evenodd" d="M 0,0 L 0,12 1.75,12 1.75,7 6,7 A 3.0,3.5 0 0,0 6,0 Z M 1.75,1.75 L 1.75,5.25 5.5,5.25 A 1.58,1.75 0 0,0 5.5,1.75 Z"/>
</svg>
```

The colour is unset, not an explicit `#000000`.